packages/foreman/nodejs-downshift/nodejs-downshift.spec

```diff
@@ -0,0 +1,92 @@
+%{?scl:%scl_package nodejs-%{npm_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global npm_name downshift
+
+Name: %{?scl_prefix}nodejs-downshift
+Version: 5.4.7
+Release: 1%{?dist}
+Summary: A set of primitives to build simple, flexible, WAI-ARIA compliant React autocomplete components
+License: MIT
+Group: Development/Libraries
+URL: https://github.com/downshift-js/downshift#readme
+Source0: https://registry.npmjs.org/@babel/runtime/-/runtime-7.10.5.tgz
```

Comment:
I think all these dependencies except `compute-scoll-into-view` are already present in our packaging. To reduce duplication I'd suggest to add that one and this package both using the `single` strategy. This keeps our packages smaller